action.yml

```diff
@@ -1,20 +1,26 @@
 name: 'Azure SQL Deploy'
-description: 'Deploy a DACPAC or a SQL script to Azure SQL database'
+description: 'Deploy a database project, DACPAC, or a SQL script to Azure SQL database'
 inputs:
   server-name:
-    description: 'Name of the Azure SQL Server name, like Fabrikam.database.windows.net.'
+    description: 'Name of the Azure SQL Server name, like Fabrikam.database.windows.net.'
     required: false
   connection-string:
-    description:  'The connection string, including authentication information, for the Azure SQL Server database.'
+    description:  'The connection string, including authentication information, for the Azure SQL Server database.'
     required: true
   dacpac-package:
-    description: 'Path to DACPAC file to deploy'
+    description: 'Path to DACPAC file to deploy'
```

Comment:
Whitespace diff.  Huh.

Reply:
In VS Code the original "spaces" were actually some invalid characters